orchestra/static/orchestra/common/components/checklist/css/checklist.scss

```diff
@@ -0,0 +1,277 @@
+$main-blue: rgb(0, 121, 191);
+$light-green: rgb(46, 204, 113);
+$dark-green: rgb(39, 174, 96);
+
+$title-height: 30px;
+$tool-width: 25px;
+$item-toolbar-width: 3 * $tool-width;
+$item-title-padding: 10px;
+$checkbox-height: $title-height;
+$checkbox-font-size: 22px;
+
+.disable-select {
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+
+@mixin vendor-prefix($name, $argument) {
```

Comment:
Apparently! Let me submit another PR reorganizing our stylesheets, I think it's time we cleaned things up.